cdap-docs/included-applications/source/etl/plugins/transforms/script.rst

```diff
@@ -72,6 +76,15 @@ schema is the same as the input schema.
           {\"name\":\"tax\",\"type\":\"double\"},
           {\"name\":\"total\",\"type\":\"double\"}
         ]
+      }",
+      "lookup": "{
+        \"purchases\":{
+          \"type\":\"DATASET\",
+          \"properties\":{
+            \"dataset_argument1\":\"foo\",
+            \"dataset_argument2\":\"bar\"
```

Comment:
It's not quite the same as http://docs.cask.co/cdap/3.3.0-SNAPSHOT/en/developers-manual/building-blocks/datasets/system-custom.html#passing-properties.

There are (1) properties you pass when creating a dataset, and (2) arguments you pass when acquiring a dataset to operate on. The page you linked to refers to (1) whereas here we are talking about (2). Not sure where (2) is documented though.